### PR TITLE
DR-2114 Expire token cache in integration tests

### PR DESCRIPTION
**Pull request by nmalfroy**

### CODE REVIEW

File: src/test/java/bio/terra/common/auth/AuthService.java [hunk 1]
```diff
@@ -33,8 +40,10 @@
   private JacksonFactory jsonFactory = JacksonFactory.getDefaultInstance();
   private File pemfile;
   private String saEmail;
-  private Map<String, String> userTokens = new HashMap<>();
-  private Map<String, String> directAccessTokens = new HashMap<>();
+  private Map<String, String> userTokens =
+      Collections.synchronizedMap(new PassiveExpiringMap<>(TimeUnit.MINUTES.toMillis(10)));
```
Comment (pshapiro4broad) [inline src/test/java/bio/terra/common/auth/AuthService.java]: There are a ton of ways to write this, I kind of like `Duration.ofMinutes(10).toMillis()`

**Review by pshapiro4broad — APPROVED** [review]
I hope this works!

File: src/test/java/bio/terra/common/auth/AuthService.java (hunk 1 above)
Comment (nmalfroy) [reply src/test/java/bio/terra/common/auth/AuthService.java]: Ah, that's good to know!

**Comment by nmalfroy** [comment]
> I hope this works!

Me too!

**Review by tlangs — APPROVED** [review]
Hope this works!

**Pull request merged**